cmd/client-daemon.go

```diff
@@ -63,7 +63,7 @@ var startDaemonCmd = &cobra.Command{
 
 		logger.Info().Msgf("daemon started at %s", time.Now().Local())
 
-		go startgRPCServer()
+		startgRPCServer()
```

Comment:
why was this removed as a goroutine? IIRC the daemon won't work otherwise because startgRPCServer is blocking

Reply:
I just forgot to re-add this. I drop go daemon whenever I want to debug rpc

Reply:
Forgot to re-add